src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -95,31 +95,161 @@
   <md-tabs flex md-border-bottom md-dynamic-height>
     <md-tab label="Pods">
       <md-content>
-        <table class="kd-replicasetdetail-table" cellspacing="0" cellpadding="15">
+        <table class="kd-replicasetdetail-table" cellspacing="0">
           <thead>
           <tr>
-            <th class="kd-replicasetdetail-table-cell">
-              Pod<i class="material-icons kd-replicasetdetail-table-icon">help</i>
+            <th class="kd-replicasetdetail-table-header">
+              <a class="kd-replicasetdetail-pointer" ng-click="ctrl.sortPods('name')">
+                <i ng-show="ctrl.arePodsSortedBy('name', true)"
```

Comment:
Documentation example https://design.google.com/icons/ uses `<i>` tag, but I agree, that we should be consistent and I will change it to `<md-icon>`.